addons/l10n_ar/i18n/es.po

```diff
@@ -6,8 +6,8 @@ msgid ""
 msgstr ""
 "Project-Id-Version: Odoo Server 13.0+e\n"
 "Report-Msgid-Bugs-To: \n"
-"POT-Creation-Date: 2020-01-15 13:31+0000\n"
-"PO-Revision-Date: 2020-01-15 13:31+0000\n"
+"POT-Creation-Date: 2020-05-14 15:18+0000\n"
+"PO-Revision-Date: 2020-05-14 15:18+0000\n"
```

Comment:
@zaoral Can you do a rebase and not touch the date?  I get conflicts with this all the time.